composer.json

```diff
@@ -63,7 +63,7 @@
         ],
         "post-install-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
-            "./vendor/bin/phpcs --config-set installed_paths ../../drupal/coder/coder_sniffer"
+            "test -e ./vendor/bin/phpcs && ./vendor/bin/phpcs --config-set installed_paths ../../drupal/coder/coder_sniffer || echo no phpcs found - installed_paths not set"
```

Comment:
I think this command will fail when executed:

`echo no phpcs found - installed_paths not set`

It should probably be

`echo 'no phpcs found - installed_paths not set'`

if what you want to output is `no phpcs found - installed_paths not set`

Reply:
The command will not fail, but your proposal looks better anyway. I fixed it.

Reply:
If I run this in my CLI:

```
echo no phpcs found - installed_paths not set
```

an error is thrown